test/state/state.hpp

```diff
@@ -24,6 +24,26 @@ class State
         assert(r.second);
         return r.first->second;
     }
+
+    Account& get(const address& addr)
+    {
+        assert(m_accounts.count(addr) == 1);
+        return m_accounts.find(addr)->second;
+    }
+
+    Account* get_or_null(const address& addr)
+    {
+        const auto it = m_accounts.find(addr);
+        if (it != m_accounts.end())
+            return &it->second;
+        return nullptr;
+    }
+
+    Account& get_or_create(const address& addr) { return m_accounts[addr]; }
```

Comment:
Hm, so will `get()` fail if the account doesn't exists? Or why do you need `get_or_create`?

Reply:
maybe this is just `operator[]` ?

Reply:
I replaced single `operator[]` with this ugly names because I want to know where new accounts are created (in the context of State cache). This already requires one more round of redesign and probably one more when state journal will be introduced.

From the current usage I need should replace this one with:
`get_or_create(address, Account{...})` because I often need to modify the created account straight away.